Facebook.Android/nuget/Xamarin.Facebook.AudienceNetwork.Android.nuspec

```diff
@@ -15,6 +15,7 @@
     <dependencies>
```

Comment:
Similar situation here as above, the extra dependencies you see in packages.config get pulled in as sub-dependencies of packages which is why they don't match.